### PR TITLE
wifi: Fix monitor sample

### PR DESCRIPTION
**Pull request by krish2718**

### CODE REVIEW

**Review by chiranjeevi2776 — APPROVED** [review]

**Review by VivekUppunda — APPROVED** [review]

File: samples/wifi/monitor/README.rst [hunk 1]
```diff
@@ -117,6 +117,15 @@ Testing
       [00:00:05.492,980] <inf> monitor:       Null Count: 0
       [00:00:05.493,011] <inf> monitor:       QoS Null Count: 0
 
+Offline net capture
+*******************
+
+The sample supports offline net capture feature in Zephyr, see `Zephyr net capture`_ for details.
+To enable this feature, use the configuration overlay files ``overlay-net-capture.conf`` and ``overlay-netusb.conf``.
```
Comment (richabp) [inline samples/wifi/monitor/README.rst:124]:
```suggestion
To enable this feature, use the configuration overlay files :file:`overlay-net-capture.conf` and :file:`overlay-netusb.conf`.
```

File: samples/wifi/monitor/README.rst [hunk 2]
```diff
@@ -117,6 +117,15 @@ Testing
       [00:00:05.492,980] <inf> monitor:       Null Count: 0
       [00:00:05.493,011] <inf> monitor:       QoS Null Count: 0
 
+Offline net capture
+*******************
+
+The sample supports offline net capture feature in Zephyr, see `Zephyr net capture`_ for details.
```
Comment (richabp) [inline samples/wifi/monitor/README.rst:123]:
```suggestion
The sample supports the offline net capture feature in Zephyr, see `Zephyr net capture`_ for details.
```

File: samples/wifi/monitor/README.rst [hunk 3]
```diff
@@ -117,6 +117,15 @@ Testing
...
+To enable this feature, use the configuration overlay files ``overlay-net-capture.conf`` and ``overlay-netusb.conf``.
+
+When the offline net capture feature is enabled, incoming IEEE 802.11 packets are routed to the offline storage over the net capture tunnel.
+They can then be analyzed using Wireshark.
```
Comment (richabp) [inline samples/wifi/monitor/README.rst:127]: Can a link to "Wireshark" be added here?
```suggestion
These packets can then be analyzed using Wireshark.
```

**Review by richabp — CHANGES_REQUESTED** [review]

**Review by richabp — APPROVED** [review]

File: samples/wifi/monitor/src/main.c [hunk 4]
```diff
@@ -66,6 +71,13 @@ typedef struct {
 	int qosDataCount;
 	int nullCount;
 	int qosNullCount;
+	int actionCount;
+	int cfEndCount;
+	int unknownMgmtCount;
+	int unknownCtrlCount;
+	int unknownDataCount;
+	int reservedFrameCount;
+	int unknownFrameCount;
```
Comment (jfischer-no) [inline samples/wifi/monitor/src/main.c:80]: Why does it not follow the coding style?

File: samples/wifi/monitor/src/main.c [hunk 5]
```diff
@@ -171,41 +214,83 @@ static void parseAndUpdateStats(unsigned char *packet, packetStats *stats)
...
 		/* Handle unknown frame types as needed */
-		LOG_INF("Unknown frame type: %d", frameType);
+		stats->unknownFrameCount++;
+		LOG_DBG("Unknown frame type: %d", frameType);
 		break;
 	}
 }
 
-static int printStats(void)
+static void printStats(void)
```
Comment (jfischer-no) [inline samples/wifi/monitor/src/main.c:239]: Fix coding style.

**Review by jfischer-no — CHANGES_REQUESTED** [review]

File: samples/wifi/monitor/src/main.c (hunk 4 above)
Comment (krish2718) [reply samples/wifi/monitor/src/main.c:80]: Do you mean not using the camel case?

Comment (jfischer-no) [reply samples/wifi/monitor/src/main.c:80]: yes

Comment (krish2718) [reply samples/wifi/monitor/src/main.c:80]: I couldn't find anything in confluence or Zephyr guidelines that camel case is disallowed, can you please point me to any link for guidelines?

**Review by sachinthegreen — APPROVED** [review]

File: samples/wifi/monitor/src/main.c (hunk 4 above)
Comment (jfischer-no) [reply samples/wifi/monitor/src/main.c:80]: You are probably familiar with the code base and have noticed that, with very few exceptions, no camelcase is used.
https://docs.zephyrproject.org/latest/contribute/guidelines.html#coding-style
https://kernel.org/doc/html/latest/process/coding-style.html
https://kernel.org/doc/html/latest/process/coding-style.html#naming

Comment (krish2718) [reply samples/wifi/monitor/src/main.c:80]: Sure, and I did go through above Zephyr links, but couldn't find any reference to `camel case`. I know its frowned upon in Linux, but wasn't sure about Zephyr.

**Pull request merged**